build.sbt

```diff
@@ -113,7 +113,7 @@ lazy val microservice = Project(appName, file("."))
   .settings(defaultSettings(): _*)
   .settings(
     Keys.fork in Test := true,
-    scalaVersion := "2.12.14",
+    scalaVersion := "2.12.15",
```

Comment:
Can you remove the `evictionWarningOptions` setting just below; SBT is warning that it's unused